### PR TITLE
Added schema for MCA LSA service station specialist finder

### PR DESCRIPTION
**Pull request by minhngocd**
https://trello.com/c/um5lWlkX/2586-mca-specialist-finder-request-life-saving-appliances

### CODE REVIEW

File: config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+        "label": "Callegari",
+        "value": "callegari"
+      },
+      {
+        "label": "Crewsaver",
+        "value": "crewsaver"
+      },
+      {
+        "label": "CSM Liferafts (Shanghai Star)",
+        "value": "csm-liferafts-shanghai star"
```
Comment (ryanb-gds) [inline config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json]: This shouldn't have a space in it

File: config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        "label": "Norsafe Boats",
+        "value": "norsafe-boats"
+      },
+      {
+        "label": "Ocean Safety (own brand)",
+        "value": "ocean-safety-own brand"
+      },
+      {
+        "label": "Ocean Safety (Remploy brand)",
+        "value": "ocean safety-remploy-brand"
```
Comment (ryanb-gds) [inline config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json]: Space

**Review by ryanb-gds — CHANGES_REQUESTED** [review]
Jeez I hate reviewing these

**Comment by lauraghiorghisor-tw** [comment]
> Jeez I hate reviewing these

Those dashes had to be changes everywhere :)

File: config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+        "label": "Norsafe Boats",
+        "value": "norsafe-boats"
+      },
+      {
+        "label": "Ocean Safety (own brand)",
+        "value": "ocean-safety-own-brand"
+      },
+      {
+        "label": "Ocean Safety (Remploy brand)",
+        "value": "ocean safety-remploy-brand"
```
Comment (ryanb-gds) [inline config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json]: Looks like there's still a space here when there should be a dash?

**Review by ryanb-gds — COMMENTED** [review]
Think we missed one 😢

File: config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json (hunk 3 above)
Comment (lauraghiorghisor-tw) [reply config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json]: Make it stop.

Comment (lauraghiorghisor-tw) [reply config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json]: I did fix a -brand one, but it was ANOTHER ONE.

**Review by ryanb-gds — APPROVED** [review]

**Pull request merged**